dd-trace-php/5.6/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM circleci/php:5.6
+
+USER root
+
+RUN apt-get update \
+    && apt-get install -y \
```

Comment:
can you additionally install Valgrind ? (its used in C extension tests) would be great to skip that step from CI